tests/FSharp.Test.Utilities/Assert.fs

```diff
@@ -1,49 +1,46 @@
 namespace FSharp.Test
 
 module Assert =
-    open FluentAssertions
     open System.Collections
     open Xunit
     open System.IO
 
+    // Equivalent, with message
     let inline shouldBeEqualWith (expected : ^T) (message: string) (actual: ^U) =
-        actual.Should().BeEquivalentTo(expected, message) |> ignore
+        try
+            Assert.Equivalent(expected, actual)
+        with e ->
+            Assert.True(false, message);
+
+    // Equivalent, with no message
+    let inline shouldBe (expected : ^T) (actual : ^U) =
+        Assert.Equivalent(expected, actual)
 
-    let inline shouldBeEquivalentTo (expected : ^T) (actual : ^U) =
-        actual.Should().BeEquivalentTo(expected, "") |> ignore
-        
     let inline shouldStartWith (expected : string) (actual : string) =
-        actual.Should().StartWith(expected) |> ignore
+        Assert.StartsWith(expected, actual)
         
     let inline shouldContain (needle : string) (haystack : string) =
-        haystack.Should().Contain(needle) |> ignore    
-
-    // One fine day, all of the 3 things below should be purged and replaced with pure Assert.Equal.
```

Comment:
The comment should stay since this todo still applies.

Reply:
The comment no longer applies.  We are now using XUnit Equivalent Apis rather than fluentassertion apis.  We will not revert the test cases to use the exact types.

Reply:
Right, yeah, that's fine then. I think ideally we should have as many exact comparisons as possible, but that's not the point of the PR.